api/src/services/users.js

```diff
@@ -1,16 +1,22 @@
 const _ = require('lodash');
 const passwordTester = require('simple-password-tester');
+const crypto = require('crypto');
 const people  = require('../controllers/people');
 const places = require('../controllers/places');
 const db = require('../db');
 const lineage = require('@medic/lineage')(Promise, db.medic);
 const getRoles = require('./types-and-roles');
 const auth = require('../auth');
+const config = require('../config');
+const phoneNumber = require('@medic/phone-number');
+const taskUtils = require('@medic/task-utils');
 
 const USER_PREFIX = 'org.couchdb.user:';
 const ONLINE_ROLE = 'mm-online';
 const DOC_IDS_WARN_LIMIT = 10000;
 
+const LOGIN_TOKEN_EXPIRE_TIME = 24 * 60 * 60 * 1000; // 24 hours
```

Comment:
Do you think 24 hours is right? I certainly wouldn't want it any longer, but I think having it shorter will result in lots of expired tokens.

Reply:
24 hours sounds about right to me. Same concerns about lots of expired tokens.